### PR TITLE
Added Chocolately and OSquery (hacktoberfest)

### PR DESCRIPTION
**Pull request by actualmermaid**
# Motivation and Context

# Description

# License Assignment

This repository is copyright 2017-2020 Joseph Block under a [Attribution-NonCommercial-ShareAlike 4.0 International](#attribution-noncommercial-sharealike-40-international) license.

- [x] I assign copyright to the changes in this PR to Joseph Block per the [Attribution-NonCommercial-ShareAlike 4.0 International](#attribution-noncommercial-sharealike-40-international) license.

# Types of changes

- [x] New entry
- [ ] Typo cleanup
- [ ] Text updates

# Checklist:

- [x] All new and existing tests pass.
- [x] List entries are sorted alphabetically.
- [x] Any list entries are one line. This makes it easier to keep the sections sorted. We let the markdown renderer handle line breaks.
- [x] I confirmed that all the links in my PR are valid.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -262,6 +262,7 @@ There are several good options:
 
 * [Ansible](http://www.ansible.com/) is designed to be minimal in nature, consistent, secure, and highly reliable. Is owned & supported by Red Hat.
 * [Chef](http://www.opscode.com/chef/) is written in Ruby and Erlang and uses a Ruby DSL to describe system configuration.
+* [Chocolatey](https://chocolatey.org/) a windows software management tool.
```
Comment (unixorn) [inline README.md:265]: Please capitalize Windows since it's a proper name in this context.

**Review by unixorn — CHANGES_REQUESTED** [review]
One minor text change, then it's good to merge. Marking it accepted for hacktoberfest.

**Comment by unixorn** [comment]
Thanks for the contribution - make that one case fix and I'll merge.

**Comment by unixorn** [comment]
Also, please check the box for the copyright assignment so I can merge it.

**Comment by actualmermaid** [comment]
Hi there! This should be fixed now. Changed it in the master!

**Comment by unixorn** [comment]
Please check the box or add a comment assigning the copyright and then I can merge. Thanks!

**Comment by unixorn** [comment]
I can't accept the PR without you acknowledging the copyright. Please add a comment accepting the copyright assignment and/or edit the PR description to check the box assigning copyright.

Thanks!

**Comment by actualmermaid** [comment]
This has been done! Thanks!

**Review by unixorn — APPROVED** [review]
LGTM

**Comment by unixorn** [comment]
Thanks, merged!

**Pull request merged**